ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -307,7 +307,7 @@ tests:
     workflow: baremetalds-e2e-virtualmedia
 - as: e2e-openstack-ipi
   steps:
-    cluster_profile: openstack-vexxhost
+    cluster_profile: openstack-vh-mecha
```

Comment:
I'm trying to understand this change vs its commit message and I don't follow.
From my understanding here, we're switching the e2e-openstack-ipi in the installer project to use mecha cloud instead of managed vexxhost?
I hope not because mecha cloud isn't ready yet (we're having performances issues).

Reply:
nevermind, I was reading the PR commit message.
Still this cloud is unstable, don't use it yet please.

Reply:
The second commit (that is now removed) was a way to test the change; the change in this PR is just preparatory.

Reply:
Seems like the commit in question (c61511319841007c9df0b02df7927050bb67ea8b) is still part of the PR. Do you still want to remove it?

Reply:
The second commit (that is NOW removed) ...... :grimacing: